java/arcs/core/data/ClaimsAndChecks.kt

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
wrt file naming: ClaimsAndChecks.kt feels kind of strange to me, especially for a file that actually defines 4 classes. I'd suggest splitting it into 4 files, but another valid option would be to find a name that adequately describes all of these concepts.